examples/modern_data_stack_assets/modern_data_stack_assets/constants.py

```diff
@@ -11,7 +11,7 @@
 
 
 def model_func(x, a, b):
-    return a * np.exp(b * (x / 10 ** 18 - 1.6095))
+    return a * np.exp(b * (x / 10**18 - 1.6095))
```

Comment:
Did the linter make this change? cc @smackesey

Reply:
Yes, change to spacing around power operator is in the new version of black we updated to.

Reply:
ah ok - will update black on my end